Tests/ProtocolBufferConversionTests.swift

```diff
@@ -0,0 +1,22 @@
+import XCTest
+@testable import XpringKit
+
+/// Tests conversion of protocol buffer to native Swift structs.
+final class ProtocolBufferConversionTests: XCTestCase {
+  func testConvertCurrency() {
+    // GIVEN a Currency protocol buffer with a code and a name.
+    let currencyCode = Data([1, 2, 3])
```

Comment:
What is `currencyCode` in the real-world? What does it signify?

Reply:
I should have filled in the `SeeAlso` above. 

> The rippled APIs support two formats of currency code for issued currencies:
> - Standard Currency Codes: As a 3-character string such as "EUR" or "USD".
> - Nonstandard Currency Codes: As a 160-bit hexadecimal string, such as "0158415500000000C1F76FF6ECB0BAC600000000". This is uncommon.


https://xrpl.org/currency-formats.html#currency-codes